@xen-orchestra/web-core/lib/components/DonutChart.vue

```diff
@@ -0,0 +1,102 @@
+<!-- v1.1 -->
+<template>
+  <div class="wrapper">
+    <svg class="donut-chart" viewBox="0 0 100 100">
+      <circle class="segment max-value-segment" cx="50" cy="50" r="40" />
```

Comment:
I think `max-value-segment` is not necessary. Just set the default color on `segment`.

```suggestion
      <circle class="segment" cx="50" cy="50" r="40" />
```